CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ if(PCAPPP_USE_SANITIZER)
 endif()
 
 # Build options (Turn on Examples and Tests if it's the main project)
-option(PCAPPP_BUILD_EXAMPLES "Build Examples" ${PCAPPP_MAIN_PROJECT})
+option(PCAPPP_BUILD_EXAMPLES "Build Examples" OFF)
```

Comment:
Keep it as it was to fix the CI

Reply:
Is this the reason CI is failing now? Also can we do this if its CI, only then use  `${PCAPPP_MAIN_PROJECT}` value ?

Reply:
Examples bin are used in the CI.

That was a decision to build the examples by default. You can disable them if you don't need them

Reply:
I have removed this commit now. Thanks for your reply